### ISSUE TITLE
Scheduled scans require redundant info

### ISSUE THREAD

**Issue by jboger** [rackerlabs/scantron#181]
Hi,

As you can see in the picture below, there are a lot of fields that one is required to fill out when adding a scheduled scan. None of the fields allow you to pick a value you have set in the other categories. You have to, for example, manually write the 'site name ID', 'scan id', 'scan command' etc, even if you have created entries for that. It would be far easier if you could choose the scan command you've created. It is also fairly redundant since even if you enter a 'scan command id', you still have to type it out in the box called 'scan command'. It doesnt seem to pull in the information from the other variables. 
This is especially problematic with the 'Targets' entry box, as that one should ideally pull in everything listed as a target in 'site name id'. If it doesnt, you have to enter every target manually in scheduled scans UI. 

So, in summary, /scantron-admin/django_scantron/scheduledscan/add/ should allow you to pick values from other variables already set. 

It is also unclear what the difference between 'scan command' and 'scan command id' is. And what the difference between 'scan start time' and 'scheduled scan start date and time' (but I'm assuming the latter is the repeated scan and the first one is a one-off). 

Hope this raised issue is somewhat clear, and that the screenshot helps to further hilight what I mean.

Thanks for an otherwise great product! I am planning on sending a few PRs your way eventually, but I just want to make sure that I am not misunderstanding some of the features first. 

![Screenshot 2020-02-27 at 09 43 05](https://user-images.githubusercontent.com/5141139/75426734-8f4c9e00-5945-11ea-9fb8-9e9f7afd9122.png)

**Comment by derpadoo** [comment]
tl;dr - Ideally, it shouldn't even be exposed through the Django admin panel.

The ScheduledScan model is populated using data from the Scan, Site, Agent, and ScanCommand models (see https://github.com/rackerlabs/scantron#database-model-graph).  I think I designed it to be standalone to reduce the complexity (no crazy foreign key, nesting, serializing, etc. issues) with making API calls and expose a simple model that could be modified (with the scan status, for example).  I need to figure out a way to enable a rescan (that changes the "Scan status" back to "pending" like I had you do) without having to mess with this model.

Appreciate the comments and looking forward to some PRs!

**Comment by derpadoo** [comment]
Going to close this one out @jboger
